test/Microsoft.ApplicationInsights.AspNetCore.Tests/JavaScript/ApplicationInsightsJavaScriptTest.cs

```diff
@@ -11,19 +13,21 @@
 
     public static class ApplicationInsightsJavaScriptTest
     {
+        private static JavaScriptEncoder encoder = JavaScriptEncoder.Create(new UnicodeRange[] { UnicodeRanges.BasicLatin });
```

Comment:
So this is just a custom implementation for [HttpUtility.JavaScriptStringEncode](https://msdn.microsoft.com/en-us/library/dd991914.aspx)? Is it recommended way?